_data/lifecycle-phases.yml

```
@@ -0,0 +1,159 @@
+phases:
+  - name: Proposal
+    description: |
+      Components under consideration for development through public discussion and a formal proposal.
+    learn_more: Visit the proposal process overview to learn more about how to contribute.
+    subphases:
+      - name: Discussion started
+        description: |
+          A component idea has been submitted to the discussion board.
+        starts_when: |
+          Anyone starts a discussion about a potential new component.
+        ends_when: |
+          The USWDS core team begins drafting a proposal for the component.
+        contribute_by:
+          - Opening a new component discussion
+          - Sharing ideas, opinions, and reactions in existing discussions
+        find_more: |
+          Find all active component discussions at the USWDS component proposal discussion board.
+      - name: Proposal in progress
+        description: |
+          The USWDS core team collects community discussion in a formal proposal for the component.
+        starts_when: The USWDS core team begins drafting the proposal for the component.
+        ends_when: The USWDS core team shares the complete proposal with the community for final feedback.
+        contribute_by:
+          - Adding missing proposal information to the discussion
+          - Adding additional information, context, or perspective to the discussion
+        find_more: |
+          **Note:** Keep all questions and comments related to the component or the proposal in the public discussion. Don't add comments in the `uswds-proposals` repo
+          </p><p>
+          Find all components with proposals in progress on the discussion board.
+      - name: Proposal open for comment
+        description: |
+          The community has a minimum of 45 days to comment on the complete proposal.
+        starts_when: The USWDS core team shares the complete proposal with the community for final feedback.
+        ends_when: The comment period closes and the core team starts evaluating feedback.
+        contribute_by:
+          - Sharing your thoughts about the proposal
+          - Asking questions about the proposal
+          - Voting for the related component discussion
+        find_more: |
+          **Note:** Keep all questions and comments related to the the proposal in the public discussion. Don't add comments in the `uswds-proposals` repo
+          </p><p>
+          Find all components with proposals open for comment on the discussion board.
```

Need to add link once it is available.
```suggestion
          Find all components with [proposals open for comment]() on the discussion board.
```